### ISSUE TITLE
Datagrid: Calendar popup issue

### ISSUE THREAD

**Issue by iamjamesreyracho** [infor-design/enterprise#6416]
**Describe the bug**
The calendar popup is unable to use in Win-Chrome and Win-Edge but it is working with Win-FF.

**To Reproduce**

1. Go to https://main-enterprise.demo.design.infor.com/components/datagrid/example-editable.html
2. Click an empty field in Order Date.
3. Click the calendar icon.

**Expected behavior**
The calendar popup should be enabled.

Win-FF (New & Classic)
https://www.screencast.com/t/Au9HzW9v 

**Version**

- ids-enterprise: v4.64.0-dev

**Screenshots**
Win-Chrome & Win-Edge (New & Classic) 
https://www.screencast.com/t/UGfNta8N  

**Platform**
 - Browser Name: Chrome (windows)

**Additional context**
NA

**Comment by tmcconechy** [comment]
Yeah that makes sense @davidcarlsonberg - good catch - maybe we can just change the example dates so the range is more current so it works (for a while) or make those dates dynamiclly set to current month - 2 +2

**Comment by iamjamesreyracho** [comment]
QA Passed. Moving this ticket to done. Thanks.
![image](https://user-images.githubusercontent.com/98729565/170431268-5b24bbb8-9e52-4b0e-9f87-0fabe4d0041f.png)
